src/components/Panel/Panel.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import { Card, Collapse } from 'react-bootstrap'
+import { Icon } from '../Icon'
+
+interface Props {
+  /** Defines the color of the panel */
+  color?: 'primary' | 'secondary' | 'success' | 'warning' | 'danger' | 'info' | 'light' | 'dark'
+  /** The body for the panel */
+  children?: React.ReactNode
+  /** The title for the Panel */
+  title?: string
+  /** The footer for the panel */
+  footer?: string
+  /** Determines if the panel can be collapsible */
+  collapsible?: boolean
+  /** Determines if the panel should be collapsed */
+  collapsed?: boolean
+}
+
+const Panel = (props: Props) => {
```

Comment:
There should be some tests for the Panel component prop.

Reply:
added